cop.h

```diff
@@ -506,14 +520,76 @@ string C<p>, creating the package if necessary.
 =cut
 */
 
+/*
+=for apidoc Am|RCPV *|RCPVx|char *pv
+Returns the RCPV structure (struct rcpv) for a refcounted
+string pv created with C<rcpv_new()>.
+
+=for apidoc Am|RCPV *|RCPV_REFCOUNT|char *pv
+Returns the refcount for a pv created with C<rcpv_new()>. 
+
+=for apidoc Am|RCPV *|RCPV_LEN|char *pv
+Returns the length of a pv created with C<rcpv_new()>. 
+
+=cut
+*/
+
+struct rcpv {
+    UV      refcount;
```

Comment:
`UV refcount` makes the reference count 64-bit on a 32-bit platform with `-Duse64bitint`, it could be STRLEN instead.

Reply:
willfix